config/environments/production.rb

```diff
@@ -64,13 +64,13 @@
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
   #devise config
-  config.action_mailer.default_url_options = { :host => 'localhost' }
+  config.action_mailer.default_url_options = { :host => Rails.application.secrets.devise_mailer_url }
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.smtp_settings = { :address => "localhost", :port => 25 }
-  config.action_mailer.default :from => 'hydranorth@mailinator.com'
+  config.action_mailer.default :from => Rails.application.secrets.devise_email
 
   #contact for config
-  config.contact_email = 'hydranorth@mailinator.com'
+  config.contact_email = Rails.application.secrets.contact_email
```

Comment:
yeah, it is referenced in here: https://github.com/ualbertalib/HydraNorth/blob/c949a558222104335a098bcd495aed726a6c25da/app/models/contact_form.rb but I don't think we use Contact Us form.